src/ray/gcs/gcs_server/test/gcs_health_check_manager_test.cc

```diff
@@ -271,17 +318,3 @@ TEST_F(GcsHealthCheckManagerTest, StressTest) {
   io_service.stop();
   t->join();
 }
-
-int main(int argc, char **argv) {
```

Comment:
Otherwise we have two main function linked in the binary, one static link another dynamic link.
The outcome of which is, `SetUp` will be called twice (confirmed via logging).